core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -224,7 +260,8 @@ public static Socket connect(Properties props, String unixPathSuffix) throws IOE
     // Gather parameters
     final String csqlInstanceName = props.getProperty(CLOUD_SQL_INSTANCE_PROPERTY);
     final boolean enableIamAuth = Boolean.parseBoolean(props.getProperty("enableIamAuth"));
-
+    // Get Creds File Location if passed
+    final String credsJsonFileLoc = props.getProperty("credsJsonFileLoc","");
```

Comment:
nit: I think something like "serviceAccountFile" might be preferable

Reply:
done